src/Action/CmdLine.hs

```diff
@@ -72,6 +73,7 @@ cmdLineMode = cmdArgsMode $ modes [search_ &= auto,generate,server,replay,test]
 search_ = Search
     {color = def &= name "colour" &= help "Use colored output (requires ANSI terminal)"
     ,link = def &= help "Give URL's for each result"
+    ,numbers = def &= name "o" &= help "Give counter for each result"
```

Comment:
Using `-o` as the short name for this seems a bit surprising. Does this need a short name?